src/integration/common/footer.js

```diff
@@ -23,5 +22,14 @@ export default () => {
       expect(brandingLinkEl.getAttribute('href')).toMatchSnapshot();
       expect(brandingImageEl).toBeInTheDocument();
     });
+
+    const footerLinks = document.querySelectorAll('footer a');
+    footerLinks.forEach(link => {
+      it(`I can see a link: ${link.textContent}`, () => {
+        expect(link).toBeInTheDocument();
+        expect(link.textContent).toBeTruthy();
+        expect(link.getAttribute('href')).toMatchSnapshot();
```

Comment:
Worth adding `expect(link.textContent).toMatchSnapshot();`?

Reply:
By putting the `link.textContent` in `it()`, it actually does appear in the snapshot: e.g. https://github.com/bbc/simorgh/pull/6282/files/3d85144c07d59299c89c895694b75c4f24c1ff86#diff-5f91e4ab11cd08e4cfe934ac2ef30167

Reply:
Phwoar this is a tidy lil two-birds-one-stone number